src/predict.rs

```diff
@@ -892,6 +915,16 @@ pub mod test {
       }
     }
 
+    let mut o8 = vec![0u8; 32 * 32];
+
+    Block4x4::pred_dc_128(&mut o8, 32, 8);
+
+    for l in o8.chunks(32).take(4) {
+      for v in l[..4].iter() {
```

Comment:
You can probably use

``` rust
assert_eq(l, &[128u8; 4][..])
```

or put the reference in a separate variable and feed `assert_eq` the two slices.

Reply:
I moved this into a test for u8 code and applied the suggestion.